renovate.json

```diff
@@ -60,7 +60,7 @@
     },
     {
       "groupName": "Yarn",
-      "matchPackageNames": ["yarn"],
+      "matchDepNames": ["yarn"],
       "extends": ["schedule:weekly"]
```

Comment:
Quick thought: surely we only need to update yarn on the default (monthly) schedule rather than anything faster?